### PR TITLE
Add localised last seen messages

### PR DESCRIPTION
**Pull request by NoobJsPerson**

### CODE REVIEW

File: src/server/controllers/memberController.js [hunk 1]
```diff
@@ -6,6 +6,7 @@
  * And we resend requests account verification emails.
  */
 
+const locale = require('date-fns/locale')
```
Comment (Naviary2) [inline src/server/controllers/memberController.js:9]: Can you merge this with the line beneath? Those imorts are from the same package.

Comment (NoobJsPerson) [reply src/server/controllers/memberController.js:9]: deconstructing doesn't work i have to work the / syntax

Comment (NoobJsPerson) [reply src/server/controllers/memberController.js:9]: probably because the package doesn't export local so its the only way to access locale

Comment (NoobJsPerson) [reply src/server/controllers/memberController.js:9]: oh its because its a sub-module thats why

**Pull request merged**